### PR TITLE
The Randomizer

### PR DESCRIPTION
**Pull request by doobiedoo23**
Ports the Mulligan syringe (which I've renamed to Randomizer here) from tgstation/-tg-station#13664. The syringe will completely randomize your appearance and name when used. It costs 1 TC. The spy bundle now includes a switchblade and a Randomizer.

### CODE REVIEW

**Comment by Chronitonity** [comment]
:+1:

**Comment by Crystalwarrior** [comment]
Isn't 1 TC a little cheap?
I mean, it basically completely negates the need for plastic surgery.

**Comment by Chronitonity** [comment]
no one does plastic surgery anyway, traitor items are allowed to negate needing something on the station.

**Comment by Crystalwarrior** [comment]
Fair point.
I already have some great plans for the toxin.

**Comment by danhibiki** [comment]
1 TC is really cheap, especially if you can use this on someone else. Go unknown stun some head, stab them with randomizer and now they look like they've stolen all their gear, for one TC

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 1]
```diff
@@ -11,6 +11,8 @@
 			new /obj/item/device/camera_bug(src)
 			new /obj/item/device/multitool/ai_detect(src)
 			new /obj/item/device/encryptionkey/syndicate(src)
+			new /obj/item/weapon/reagent_containers/syringe/mulligan(src)
+			new /obj/item/weapon/switchblade
```
Comment (Chronitonity) [inline code/game/objects/items/weapons/storage/uplink_kits.dm:15]: (src) at the end

File: code/datums/uplink_item.dm [hunk 2]
```diff
@@ -715,6 +715,13 @@ var/list/uplink_items = list()
 	cost = 5
 	surplus = 90
 
+/datum/uplink_item/stealthy_tools/mulligan
```
Comment (Chronitonity) [inline code/datums/uplink_item.dm:718]: If you wanna rename it might as well rename all the spots where it says mulligan to randomizer

**Comment by SplergeBucket** [comment]
i think the 1TC price is yeah really cheap
iunno 2 or 3  i'd say as no one's gunna find you after you randomise and swap around yer equipment or someone elses
probably best for just 2, anything else and no one's gunna buy it
otherwise i give this my stamp of aproval

**Comment by danhibiki** [comment]
For the interim, I'll merge this and we can discuss TC cost changing

**Comment by TDcoolguy300** [comment]
Goddamn it HIbiki, it will break shit now, a (src) was not added

**Comment by danhibiki** [comment]
are you serious

**Comment by Chronitonity** [comment]
I got it

**Pull request merged**